### DISCUSSION TITLE
[BUG] Commit messages are rendered with emoji

### DISCUSSION THREAD

**Discussion by Lanchon** [community/community#16133] [Other Feature Feedback, Questions, & Ideas]
When viewed via the github web interface, commit messages are rendered with emoji.

This is an abomination: commit messages predate and will outlive github.
I suppose this is a rendering bug. If it is a "feature", it is entirely out of place; please kill it.

Example: github rendering screwing up MAC addresses: https://github.com/openwrt/openwrt/commit/f8b0010dfb548469686049f85076fd6a3a6bca2e

**Comment by queenofcorgis** [comment]
Hi there! Thanks for flagging - to make sure we understand your request are you saying that text in your commit messages are being replaced by emojis?

**Comment by Lanchon** [comment]
hi, thanks!

yes. for instance in the link i posted earlier, an ethernet OID (the part of a MAC address that is registered to a specific organization) is incorrectly rendered with :ab: emoji.

**Reply by queenofcorgis** [reply]
That's what I thought, but I wanted to be 100% - thanks for confirming - I labeled this as a bug.

**Reply by Lanchon** [reply]
excuse the tone of the OP. i honestly thought no one would ever read it. thanks!

**Reply by queenofcorgis** [reply]
@Lanchon no need to worry I understand you may have been a bit frustrated!
